src/core/utils.ts

```diff
@@ -7,13 +7,9 @@ import * as riot from 'riot';
 
 export { clone, deepAssign, log, riot, GbTracker };
 
-export const WINDOW = {
-  addEventListener: (event, cb) => window.addEventListener(event, cb),
-  Image: () => new Image(),
-  document: () => window.document,
-  location: () => window.location,
-  history: () => window.history
-};
+export function WINDOW() {
```

Comment:
I think this could be simpler as a one-liner `export const WINDOW = () => window;`